developer-docs/latest/deployment/21yunbox.md

```diff
@@ -0,0 +1,66 @@
+# 21YunBox
+
+This guide explains how to update an existing Strapi project so it can be deployed on [21YunBox](https://www.21yunbox.com).
+
+21YunBox provides a Chinese CDN, continuous deployment, one-click HTTPS and [other services such as managed databases and backend web services](https://www.21yunbox.com/docs/#/), allowing to launch web projects in China.
+
+With persistent disks and managed PostgreSQL databases, 21YunBox gives you multiple different ways to store your content. 21YunBox services come with fully managed SSL, so it's no longer necessary to set up a proxy server to secure your Strapi app. Since 21YunBox services are automatically restarted if they become unresponsive, you don't need to use a process manager like `pm2` either.
+
+::: tip
+For more information consult [21YunBox's Deploy Strapi guide](https://www.21yunbox.com/docs/#/deploy-strapi)
+:::
+
+
+## Prerequisites
+
+This guide assumes you already have a Strapi project to deploy. If you need a project, use the [Quick Start](/getting-started/quick-start) to get started or fork 21YunBox's Strapi Examples:
```

Comment:
Can you make sure you put the .md on this or it will cause bad links

Reply:
updated the broken link.